example-schema/string-patterns.yaml

```diff
@@ -2,7 +2,7 @@ NODE_TYPE: '^(?=.{2,64}$)([A-Z][a-z]+)+$' # NodeType
 CODE: '^(?=.{2,64}$)[a-z0-9]+(?:-[a-z0-9]+)*$' # system-code.1
 RELATIONSHIP_NAME: '^(?=.{2,64}$)[A-Z][A-Z_]*[A-Z]$' # HAS_A_RELATIONSHIP
 ATTRIBUTE_NAME: '^(?=.{2,64}$)[a-z][a-zA-Z\d]+$' # attributeName
-COST_CENTRE: '^(?=.{2,64}$)[A-Z][A-Z]\d\d\d' # XT100
+COST_CENTRE: '^(?=.{2,64}$)[A-Z][A-Z]\d\d\d$' # XT100
```

Comment:
most of these patterns can actually be removed from this repo - an artefact from when the schema was copied over from Biz Ops Schema. 

I'll comment on each line (including this one) that can be removed, but there might be more
```suggestion
```

Reply:
actually, I can't comment on anythnig hat's more than 5 lines away from your changes, but repo name, package name and the max length ones can definitely go. Probably everything apart from the top 4 lines, so try searcing in the `example-schema` folder to see if any of the others are mentioned. If not, you can delete them